beluga_tutorial/params/tutorial.yaml

```diff
@@ -0,0 +1,14 @@
+map_size: 100 # Size of the map in [m]
+number_of_doors: 15
+number_of_particles: 300 # Fixed number of particles
+number_of_cycles: 100
+initial_pose: 0.0 #[m]
+initial_pose_sd: 20.0
+dt: 1.0 #[s]
+velocity: 1.0 #[m/s]
+translation_sd: 1.0
+sensor_range: 3.0 # Sensor range of view in [m]
+sensor_model_sigam: 1.0
+min_particle_weight: 0.08
+dataset_path: "./src/beluga/beluga_tutorial/bags/dataset.yaml"
```

Comment:
This path here depends on the workspace directory layout, and only works with that specific layout. Running this script from `/ws` in our development container.

Reply:
In the ideal case, I should create the package outside of the beluga /ws. In general, I need to use beluga as an external dependency. So, actually, I should clarify to the users in the tutorial that they should choose their own path.

Reply:
How about we create the data set file in the current working directory from where this program was run?

Reply:
Yes it also an option, just I don't found it confortable since for now I'm running the tutrial from the install/ folder, but I can try.

Reply:
Can't you use `ros2 run beluga_tutorial ...` from everywhere?

Reply:
No, apparently I need to use ament to be able to run the code using ros2